.github/workflows/release-code-freeze.yml

```
@@ -226,3 +226,55 @@ jobs:
+            hmac.update(requestBody);
+            const signature = hmac.digest('hex');
+            
+            const response = await fetch(process.env.WPCOM_RELEASE_WEBHOOK_URL, {
+              method: 'POST',
+              headers: {
+                'Content-Type': 'application/json',
+                'X-Hub-Signature-256': `sha256=${signature}`
+              },
+              body: requestBody
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)